contracts/SnappAuction.sol

```diff
@@ -26,7 +26,20 @@ contract SnappAuction is SnappBase {
     mapping (uint16 => StandingOrderData) public standingOrders;
 
     uint public auctionIndex = MAX_UINT;
-    mapping (uint => SnappBaseCore.PendingBatch) public auctions;
+
+    struct PendingAuction {
+        address solver;
+        uint objectiveValue;           // Traders utility
+        bytes32 solutionHash;          // Succinct record of trade execution & prices
+        uint solutionAcceptedTime;   // Time solution was accepted (written at time of solutionHash)
```

Comment:
How about fullSolutionRevealedTime?

nit: Are the variable collated in some meaningful way? We could order it in the order that they are used (1st all order related stuff, then solution bidding, then solution applied)?

Reply:
Unfortunately, this is not the `fullSolutionRevealed` time because of the second phase. There is a difference between when the solution is revealed and when the solution is accepted. For more information on this, please refer to the discussion #151 (specifically the most recent post about `fallbackApplyAuction`).

Yes, I will try and order the variables better.